Modules/Sources/WordPressUI/Extensions/UITableView+Helpers.swift

```diff
@@ -87,4 +87,29 @@ extension UITableView {
         }
         return true
     }
+
+    /// Updates the layout of the given state view inside the table view to
+    /// achieve a nice parallax effect and ensure the state view covers the
+    /// visible scroll view area.
+    public func layoutEmptyStateView(_ stateView: UIView, in viewController: UIViewController) {
```

Comment:
I haven't integrated it yet, but I plan to use it in `ReaderCommentsViewController` together with the new `EmptyStateView`.